CHANGELOG.md

```diff
@@ -6,6 +6,53 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.8.0] - 2021-11-25
+### Added
+* Overview of [new features in v0.8](docs/source/whatsnew_0_8.md)
+* Network modules for differentiable neural network topology search (DiNTS)
+* Multi-instance learning transforms and models for digital pathology image analysis
```

Comment:
change to"

Multiple Instance Learning transforms and models for digital pathology WSI analysis